src/tools/cephfs/cephfs-shell

```diff
@@ -615,6 +614,13 @@ class CephFSShell(Cmd):
         """
         Copy a local file/directory to CephFS.
         """
+        if not os.path.isfile(args.local_path):
+            if not os.path.isdir(args.local_path):
```

Comment:
Better to write a single `if` than 2?

```suggestion
        if not os.path.isfile(args.local_path) and not os.path.isdir(args.local_path):
```

Or -

```suggestion
        if not (os.path.isfile(args.local_path) or os.path.isdir(args.local_path)):
```

Reply:
Okay

Reply:
I've made the suggested changes, thanks!